src/accelerate/utils/constants.py

```diff
@@ -37,9 +37,11 @@
 FSDP_AUTO_WRAP_POLICY = ["TRANSFORMER_BASED_WRAP", "SIZE_BASED_WRAP", "NO_WRAP"]
 FSDP_BACKWARD_PREFETCH = ["BACKWARD_PRE", "BACKWARD_POST", "NO_PREFETCH"]
 FSDP_STATE_DICT_TYPE = ["FULL_STATE_DICT", "LOCAL_STATE_DICT", "SHARDED_STATE_DICT"]
+FSDP2_STATE_DICT_TYPE = ["SHARDED_STATE_DICT"]
 FSDP_PYTORCH_VERSION = (
     "2.1.0.a0+32f93b1"  # Technically should be 2.1.0, but MS-AMP uses this specific prerelease in their Docker image.
 )
+FSDP2_PYTORCH_VERSION = "2.5.1"
```

Comment:
Should we make this `2.6.0`? 

Since imports such as  https://github.com/huggingface/accelerate/blob/d7c741a6bc575878ccafcbdc957cc8fcb5ba5c34/src/accelerate/utils/dataclasses.py#L1987 would break for any version below 2.6.0